buildpipeline/pipeline.json

```diff
@@ -147,6 +147,32 @@
       "DependsOn": [
         "Trusted-All-Release"
       ]
+    },
+    {
+      "Name": "Signing validation",
+      "Parameters": {
+        "TreatWarningsAsErrors": "false"
+      },
+      "BuildParameters": {
+        "BuildConfiguration": "Release"
+      },
+      "Definitions": [
+        {
+          "Name": "Core-Setup-Signing-Validation",
```

Comment:
Does this need to be named `Core-Setup`?  That seems redundant with the repo's name.